panther_audit_rules/panther_detection_deleted.py

```diff
@@ -0,0 +1,35 @@
+from panther_base_helpers import deep_get
+
+PANTHER_DETECTION_DELETE_ACTIONS = [
+    "DELETE_DATA_MODEL",
+    "DELETE_DETECTION",
+    "DELETE_DETECTION_PACK_SOURCE",
+    "DELETE_GLOBAL_HELPER",
+    "DELETE_LOOKUP_TABLE",
+    "DELETE_SAVED_DATA_LAKE_QUERY",
+]
+
+
+def rule(event):
+    return (
+        event.get("actionName") in PANTHER_DETECTION_DELETE_ACTIONS
+        and event.get("actionResult") == "SUCCEEDED"
+    )
+
+
+def title(event):
+    detections_list = deep_get(event, "actionParams", "input", "detections")
+    deleted_detections_string = " ".join([x.get("id") for x in detections_list])
+    return (
+        f"Detection Content has been deleted by {event.udm('actor_user')}: "
+        f"{deleted_detections_string}"
```

Comment:
Does it make sense to leave the detection name out of the title? You also have it in the alert context, it seems like this could potentially create a large title string if many detections were deleted at the same time?

Reply:
Thats a good call, changing